### DISCUSSION TITLE
blacklist url with arguments

### DISCUSSION THREAD

**Discussion by hdez-es** [DigitalRuby/IPBan#219] [Q&A] [answered]
Hello

I want to know if I can use blacklist or whitelist url with arguments
domain.com/ipban.php?t=blacklist
domain.com/ipban.php?t=whitelist

I'm trying but it not works
if i use 2 different urls it works ok

**Comment by jjxtra** [comment]
I'm trying the Blacklist property out with 2 urls (same domain, different query string), separated by comma. They both are pulling the ip addresses correctly.

When you say it doesn't work, is it not pulling down any ip addresses? Is it ignoring one of the urls?

**Comment by hdez-es** [comment]
if I configure the ipban.config like this
\<add key="Whitelist" value="domain.com/ipban.php?t=whitelist"/>
\<add key="Blacklist" value="domain.com/ipban.php?t=blacklist"/>
rules are not created in the firewall

Instead if I configure it like this
\<add key="Whitelist" value="domain.com/ipban_whitelist.php"/>
\<add key="Blacklist" value="domain.com/ipban_blacklist.php"/>
the rules are created

**Reply by jjxtra** [reply]
Can you email support@ipban.com with your exact urls with the query string? I can debug them.

**Comment by jjxtra** [comment, marked as answer]
The ? is a comment character, so you must use a 302 redirect url without a ? char if you must have a ? in the url. Apologies for the inconvenience.

**Comment by hdez-es** [comment]
Ok. Thanks
